README.md

```diff
@@ -96,6 +96,9 @@ The user that will be created using the seeder will have the following credentia
 - Email: **admin@admin.com**
 - Password: **admin**
 
+### CORS:
+It's highly recommended that you add the [CORS middleware](https://docs.vapor.codes/2.0/http/cors/) to your server.
```

Comment:
Would it more precise to say project instead of server?

Reply:
In this case I meant server as in Droplet (which is a server), but I'll change it to project.